src/docs/sdk/envelopes.mdx

```diff
@@ -593,14 +593,17 @@ SDKs should not rely on Envelope header authentication to retain backward compat
 Event ingestion imposes limits on the size and number of Items in Envelopes.
```

Comment:
I know you didn't touch this but if we're already lowercasing:
```suggestion
Event ingestion imposes limits on the size and number of items in envelopes.
```

Reply:
Actually, the whole file could use some lowercase treatment but I can't add suggestions further up. 🙈 I can make a PR to do this separately.

Reply:
Ok I've read the file and I guess the intention is to define envelopes, items etc. as concepts, in that case I think the capital first letter is fine.